src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1775,6 +1781,34 @@
                         </UserData>
                     </template>
 
+                    <template tid="WaitHandleWaitStart">
+                        <data name="WaitSource" inType="win:UInt8" map="WaitHandleWaitSourceMap" />
+                        <data name="ClrInstanceID" inType="win:UInt16" />
```

Comment:
Since this is a new event, ClrInstanceID could be moved to be the first parameter so that the other relevant parameters are together.